src/player.cpp

```diff
@@ -4835,3 +4960,39 @@ void Player::updateRegeneration()
 		condition->setParam(CONDITION_PARAM_MANATICKS, vocation->getManaGainTicks() * 1000);
 	}
 }
+
+bool Player::addLoyalty(int32_t points)
+{
+	if (loyaltyPoints == 0) {
+		return false;
+	}
+
+	loyaltyPoints += points;
+	return true;
+}
+
+std::string Player::getLoyaltyTitleDescription() const
+{
+	if (loyaltyPoints <= 50) {
+		return "a Scout of Tibia";
```

Comment:
maybe we should use world name or citizen instead of tibia

Reply:
okay, I'll use ConfigManager::SERVER_NAME